tests/ci/labels/features.go

```diff
@@ -4,16 +4,57 @@ import (
 	. "github.com/onsi/ginkgo/v2"
 )
 
-var FeatureCluster = Label("feature-cluster")
-var FeatureRoles = Label("feature-roles")
-var FeatureIDP = Label("feature-idp")
-var FeatureIngress = Label("feature-ingress")
-var FeatureKubeletConfig = Label("feature-kubeletconfig")
-var FeatureMachinepool = Label("feature-machinepool")
-var FeatureNetworkVerifier = Label("feature-networkverifier")
-var FeatureOidcConfig = Label("feature-oidcconfig")
-var FeatureOidcProvider = Label("feature-oidcprovider")
-var FeatureRegion = Label("feature-region")
-var FeatureUser = Label("feature-user")
-var FeatureToken = Label("feature-token")
-var FeatureVersion = Label("feature-version")
+// feature labels
```

Comment:
Can you add a comment how this list is constructed ?
That would help to maintain it :)

Reply:
Sure.

Reply:
Please add a description here !